simbad/util/__init__.py

```diff
@@ -33,6 +35,16 @@ def get_sequence(input_f, output_s):
             f_out.write(seq_info[i] + os.linesep)
 
 
+def get_mrbump_ensemble(mrbump_dir, final):
+    """Output ensemble from mrbump directory to a dat file"""
+    if os.path.isdir(mrbump_dir):
+        ensemble = glob.glob(os.path.join(mrbump_dir, 'models', 'domain_*', 'ensembles',
```

Comment:
If these are many files and you only provide them to the `convert_pdb_to_dat` function, you should probably use `glob.iglob` which returns a generator. Thus you only need to iterate over the files once.